share/templates/08-Orchestral/02-Symphony_Orchestra.mscx

```diff
@@ -1,12 +1,10 @@
-<?xml version="1.0" encoding="UTF-8"?>
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why is this happening? Encoding style changed?

Reply:
A trailing space got lost it seems? Which is fine.